core/lib/dal/src/storage_logs_snapshots_dal.rs

```diff
@@ -0,0 +1,117 @@
+use crate::StorageProcessor;
+use zksync_types::snapshots::{SnapshotFactoryDependency, SnapshotStorageLog};
+use zksync_types::{AccountTreeId, Address, L1BatchNumber, MiniblockNumber, StorageKey, H256};
+
+#[derive(Debug)]
+pub struct SnapshotChunksDal<'a, 'c> {
+    pub(crate) storage: &'a mut StorageProcessor<'c>,
+}
+
+impl SnapshotChunksDal<'_, '_> {
+    // not yet used by snapshot_creator
+    pub async fn get_last_miniblock_number(
```

Comment:
Nit: There's already `get_miniblock_range_of_l1_batch()` in `BlocksDal`. I'd suggest to reuse it, although technically speaking it does some unnecessary work.